torch/csrc/jit/function_schema.h

```diff
@@ -18,8 +18,8 @@ struct Argument {
       bool kwarg_only = false)
       : name(std::move(name)),
         type(type? type : DynamicType::get()),
-        N(N),
-        default_value(default_value),
+        N(std::move(N)),
```

Comment:
Are you seriously moving `int32_t` around? 😄

Reply:
Yeah I think clang tidy only sees the fact that `optional` is not a primitive type, even though it will only store primitive types in its implementation